src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +51,45 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
```

Comment:
Ensure that `yesterdayMovieSession` has been assigned an ID after being added to `movieSessionService`. Otherwise, calling `getId()` may result in a `NullPointerException`.